### ISSUE TITLE
Implement `selecting { table.* }`

### ISSUE THREAD

**Issue by rzane** [rzane/baby_squeel#70]
```ruby
Post.joins(:author).selecting { [post.id, author.*] }
```

**Comment by octavpo** [comment]
Yes I'd very much like to see this implemented and finally be able to get rid of all the strings I have to use now. Thanks.

**Comment by rzane** [comment]
Yeah, this is definitely going to happen. Should be pretty trivial to implement if you'd be interested in taking a stab at it.

The gist of it is:

```
def *
end
```
